libs/platform/experience/entity-text/entity-text.component.ts

```diff
@@ -0,0 +1,26 @@
+import { EntityService } from '@spryker-oryx/core';
```

Comment:
I have a circular deps error in vscode on this line

Reply:
hmm, not sure, but there was one in other place, probably fixed